tests/integration/buildcmd/test_build_in_source.py

```diff
@@ -0,0 +1,49 @@
+import os
+import pytest
+import logging
+from unittest import skip
+from parameterized import parameterized
+
+from tests.integration.buildcmd.build_integ_base import BuildIntegProvidedBase
+
+LOG = logging.getLogger(__name__)
+
+
+@skip("Building in source option is not exposed yet. Stop skipping once it is.")
```

Comment:
currently skipping as the click option for building in source is not exposed yet (so this test would fail)

Reply:
btw, made sure it passed when not skipping if the click option was exposed
![Screen Shot 2023-02-09 at 1 44 18 PM](https://user-images.githubusercontent.com/86501267/217946115-e42371a6-6fcd-40d7-85b9-26bdfce89919.png)